app.js

```diff
@@ -20,6 +21,53 @@ const auth = authorization(app);
 app.use(auth.initialize());
 app.auth = auth;
 
+let detailsRoute = function(req, res, next) {
```

Comment:
You shouldn't be using `let` if the value is not modified.
Use `const` instead.

Also if you are not using arrow functions you don't need to assign a function to a variable 
```Javascript
let detailsRoute = function(req, res, next) {

```

Instead declare the function directly. 

```Javascript
function detailsRoute(req, res, next) {
```

Or if you really want to assign it to a variable use arrow functions

```Javascript
let detailsRoute = (req, res, next) => {
```